spec/index.bs

```diff
@@ -7014,6 +6997,114 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
         or the third item of the sequence.
 </div>
 
+# Feature Definitions # {#feature-definitions}
+
+The capabilities of each [=feature=] approved by the WebGPU Working Group are defined here.
+
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>depth-clamping</dfn> ## {#depth-clamping}
+
+Issue: Define functionality when the {{GPUFeatureName/"depth-clamping"}} [=feature=] is enabled.
+
+**Feature Dictionary Values**
+
+The following dictionary values are supported if and only if the {{GPUFeatureName/"depth-clamping"}}
+[=feature=] is enabled, otherwise they are ignored:
```

Comment:
It shouldn't be ignored, it should be validated (e.g. in this case, it MUST be false if the feature is not enabled)